applications/plugins/SofaPython/Binding_BaseContext.cpp

```diff
@@ -71,6 +71,21 @@ extern "C" PyObject * BaseContext_getRootContext(PyObject *self, PyObject * /*ar
     return sofa::PythonFactory::toPython(obj->getRootContext());
 }
 
+void SofaPythonListToStdStr(PyObject *list, std::stringstream& out)
```

Comment:
a comment on the function would be welcome! :p

no need for c++, you can already use SofaPython.tools.listToStr() to do the same.